generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs

```diff
@@ -706,4 +727,77 @@ public class CacheConfiguration<% if (cacheProvider === 'hazelcast') { %> implem
         return defaultCache;
     }
     <%_ } _%>
+
+    <%_ if (cacheProvider === 'redis') { _%>
+    private final javax.cache.configuration.Configuration<Object, Object> jcacheConfiguration;
+
+        public CacheConfiguration(JHipsterProperties jHipsterProperties) {
+            MutableConfiguration<Object, Object> jcacheConfig = new MutableConfiguration<>();
+            Config config = new Config();
+            config.useSingleServer()
+                .setAddress(jHipsterProperties.getCache().getRedis().getServer())
+                .setSubscriptionConnectionMinimumIdleSize(1)
```

Comment:
As we use all the default values, could we omit all these configuration options?

Reply:
Thanks for your review :+1: